org.OpenT2T.Sample.SuperPopular.Thermostat/Wink Thermostat/v2/package.json

```diff
@@ -0,0 +1,42 @@
+{
+  "name": "opent2t-translator-wink-thermostat",
+  "version": "1.0.0",
+  "description": "Wink Thermostat translator",
+  "author": "OpenT2T",
+  "license": "MIT",
+  "scripts": {
+    "test": "ava"
+  },
+  "ava": {
+    "files": [
+      "tests/*.js"
+    ]
+  },
+  "dependencies": {
+    "opent2t": "*",
+    "opent2t-onboarding-winkhub": "*",
+    "opent2t-translator-helper-wink": "1.0.0"
+  },
+  "devDependencies": {
+    "delay": "^1.3.1",
+    "jshint": "^2.9.2",
+    "ava": "^0.15.2"
+  },
+  "opent2t": {
+    "interfaces": {
+      "org.OpenT2T.Sample.SuperPopular.Thermostat": {
```

Comment:
>org.OpenT2T.Sample.SuperPopular.Thermostat [](start = 7, length = 42)

This is OK, but I was only suggesting to change the .xml file name, not necessarily the .js file name. The interface JavaScript file name (module name) can be simply "thermostat.js". The module name only needs to be unique within the package, not globally unique like the interface name.